addons/backups-restic/backups-restic.yaml

```diff
@@ -52,7 +52,7 @@ spec:
               path: /etc/kubernetes/pki
           initContainers:
           - name: snapshoter
-            image: {{ Registry "registry.k8s.io" }}/etcd:3.5.4-0
+            image: {{ Registry "gcr.io" }}/etcd-development/etcd:v3.5.6
```

Comment:
Why change the image? The original image is used by kubeadm for etcd.

Reply:
It's a "draft" PR. Anyways, `registry.k8s.io/etcd:3.5.4-0` doesn't have a shell, does it? Did we test these changes https://github.com/kubermatic/kubeone/pull/2579? For me it errors out with `mkdir command not found`.